CMake/SlicerCPack.cmake

```diff
@@ -356,9 +356,18 @@ if(CPACK_GENERATOR STREQUAL "NSIS")
 
   set(_nsis_install_root "${Slicer_CPACK_NSIS_INSTALL_ROOT}")
 
+  # Use ManifestDPIAware to improve appearance of installer
```

Comment:
To simplify, include comment and also add exactly two spaces to match indentation in template, consider doing the following:

```cmake
string(APPEND CPACK_NSIS_DEFINES "\n  ;Use ManifestDPIAware to improve appearance of installer")
string(APPEND CPACK_NSIS_DEFINES "\n  ManifestDPIAware true")
```